tox.ini

```diff
@@ -78,3 +78,38 @@ commands =
     pylint -j 0 --rcfile=pylintrc securesystemslib tests
     bandit --recursive securesystemslib --exclude _vendor
     mypy
+
+# Requires docker running
```

Comment:
is this why the test is not part of the default set?

Reply:
Yes. I thought, we shouldn't require devs to run docker just for this. And it takes quite a while to start the container, especially the first time. 

On GitHub, there doesn't seem to be any default caching. Maybe, we could improve this, by pulling the images explicitly with docker or maybe by using the localstack GitHub Action. Right now docker stuff happens behind the scenes by running the `localstack` command from within tox, which has the advantage that we can run the same `tox` command locally and on GitHub.